scriptworker_client/src/scriptworker_client/github_client.py

```diff
@@ -137,7 +137,15 @@ async def get_files(self, files: Union[str, List[str]], branch: Optional[str] =
         str_query = query.substitute(owner=self.owner, repo=self.repo, fields=",".join(fields))
 
         contents = (await self._client.execute(str_query))["repository"]
-        return {k.replace(sentinel_dot, ".").replace(sentinel_slash, "/").replace(sentinel_dash, "-"): v["text"] for k, v in contents.items()}
+        ret: Dict[str, Union[str, None]] = {}
```

Comment:
nit: I'm not opposed to annotating this, but it's not at any interface boundary and type checkers would have no problem inferring the type.. so I think annotating this variable is unnecessary.

Reply:
Hm, I added this because I thought I saw a complaint about it...but maybe I conflated it with a complaint about the return value of the function. I'll see about removing it.

Reply:
Yeah:
```
[task 2025-04-14T14:47:20.837Z] src/scriptworker_client/github_client.py:140: error: Need type annotation for "ret" (hint: "ret: dict[<type>, <type>] = ...")  [var-annotated]
```